Bootstraps/AsyncInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PHPPM\Bootstraps;
+
+
+use React\EventLoop\LoopInterface;
+
+interface AsyncInterface
```

Comment:
I guess this question was raised as this interface is not used in any of the files of this PR.